.github/workflows/push-to-stage.yml

```diff
@@ -0,0 +1,44 @@
+name: Push to Staging
+
+on: 
+  push:
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Extract branch name
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      id: extract_branch
+    # Github uses different branch var names for push vs PR event types
+    - name: Extract branch name (PR Only)
+      if: github.event.action == 'opened'
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_HEAD_REF#refs/heads/})"
+      id: extract_branch_pr
```

Comment:
Will this work for cross-fork PRs? What if the same branch name exists on multiple forks with open PRs? Could we not use something guaranteed to be mutually exclusive, like PR id, instead?

Reply:
As discussed outside contributors won't be able to run these workflows so I believe this is a non issue re: branch name conflicts.